frontend/src/components/App/themeSlice.ts

```diff
@@ -149,19 +149,19 @@ const themeSlice = createSlice({
 });
 
 export const useAppThemes = () => {
```

Comment:
```suggestion
export const useAppThemes = (): AppTheme[] => {
```

there's an AppTheme type in scope you can use here for the return type, then we won't need `any` in other files